src/faebryk/libs/units.py

```diff
@@ -14,6 +14,7 @@
 
 UnitsContainer = _UnitsContainer | str
 Quantity = P.Quantity
+Unit = P.Unit
```

Comment:
This is different than above. The types are not the same.
Got to be careful here, this might break stuff. Check where Unit is used atm.